styles/Vocab/Plone/accept.txt

```diff
@@ -1,10 +1,22 @@
+async
+api 
```

Comment:
Vale does not warn about acronyms when they are uppercased, such as `API`. `api` is neither a word nor an acronym because of its improper casing, thus we want this term in the `reject.txt` file.

```suggestion
```